monitors/test/ContractMonitor.js

```diff
@@ -346,6 +349,10 @@ contract("ContractMonitor.js", function(accounts) {
     assert.isTrue(lastSpyLogIncludes(spy, "failed")); // the disputed was not successful based on settlement price
     assert.isTrue(lastSpyLogIncludes(spy, `https://etherscan.io/tx/${txObject1.tx}`));
 
+    // Advance time so that price request is for a different timestamp.
+    const nextLiquidationTimestamp = liquidationTime + 1;
+    await emp.setCurrentTime(nextLiquidationTimestamp.toString());
```

Comment:
It is neccessarty to move time forward in the EMP after the first liquidation in order to request a price for a different timestamp